data.json

```diff
@@ -30,7 +30,7 @@
         "issueNumber": 15
     },    
     {  
-        "name": "Atinuke Kayode",
+        "name": "Bolaji Ayodeji",
```

Comment:
From the screenshot image, the name is correct, the username is incorrect

Reply:
I have made the changes